### DISCUSSION TITLE
How can to utilize this resource?

### DISCUSSION THREAD

**Discussion by Martyr-23** [kinkofer/FightClub5eXML#190] [Q&A]
I ask as I've recently started using FightClub5e and found this, but as someone not familiar to GitHub or coding, how can I get the XML files to the FightClub5e app or even the GameMaster5e?

**Comment by kinkofer** [comment]
As the usage documentation states, you can't use the files here directly,
you have to build the compendium xml files, then import them into the app.
There might be a [tool somewhere](https://github.com/kinkofer/FightClub5eXML/issues/157#issuecomment-878412356) to help you do that without needing a tech background.

**Reply by Martyr-23** [reply]
Understood, thanks for the heads up.
